packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -107,10 +359,36 @@ export default decorate([
       },
     },
     computed: {
-      filteredTaskLogs: (
-        { defaultFilter, filter: value = defaultFilter },
-        { log = {} }
-      ) =>
+      log: (_, { log }) => {
+        if (log === undefined) {
+          return {}
+        }
+
+        if (log.tasks === undefined) {
+          return log
+        }
+
+        const newLog = cloneDeep(log)
+        newLog.tasks.forEach(task => {
+          if (task.tasks === undefined || get(() => task.data.type) !== 'VM') {
+            return
+          }
+          task.tasks.forEach(subTask => {
+            const isFull = get(() => subTask.data.isFull)
+            if (isFull !== undefined) {
+              task.isFull = isFull
+              return false
+            }
+          })
```

Comment:
Can't we run this loop first on `log` and `cloneDeep` it only if we actually found a `isFull`?